new_docs/DOCSTRUCTURE.md

```diff
@@ -0,0 +1,53 @@
+# doc-structure
+this is a repository for the new documentation structure
+
+
+-----------------------------------------------------------------------------
+
+
+Chapter 1 Concepts of thin-edge.io 
+
+(In this chapter terminology and concepts of thin-edge.io should be explained. Not only text should be used but also (simple/schematic) images. See the cumulocity concepts guide for examples. What we have now is too developer oriented, concepts should be more general described
```

Comment:
Having images in a documentation is a common thing. What are the reasons not having images in the repo? The only option I see is to move the doc to another repo (what we actually did but requested to add it to this repo).

Reply:
It blows up the repository size insanely fast, that's why I really don't like it. Cloning a repository that's 500MB just because there are hundreds of images (or other binary files such as .pdf or similar) drives me crazy. And that's not "hundreds of images" as in there are hundreds of images in the repository, but hundreds of _versions_ of maybe even a single image! git is bad with binary files!

In this case here I find it especially concerning, as images in documentation are rather likely to change ("rather" may be in the eye of the beholder here). The thin-edge logo is also checked into this repository, but this won't change that often. Documentation images will change far more often than the logo.

---

A possible solution would be to have the images just served from a static web space (`static.thin-edge.io` or whatever) and include them from there. They could even be organized in a seperate git repository (only the images, not the whole documentation of course), if versioning is a concern with these images. But slowing down the thin-edge.io repository (not only in download/`git-clone` speed, but also in `git-checkout` speed) for everyone because of images in the documentation sounds not nice to me.

---

Either way, I am not qualified to decide this. @didier-wenzek and @TheNeikos are. The above is just my 2ct on the topic of binary files in repositories.

Reply:
About the images. SVG diagrams or ASCII art can be really helpful. But I would really avoid JPG or PNG images. Because the content of such images is not searchable and GUI screenshots are difficult to maintain.